Resources/Prototypes/Body/Organs/dwarf.yml

```diff
@@ -1,22 +1,22 @@
 ﻿- type: entity
-  id: OrganDwarfHeart
   parent: OrganHumanHeart
+  id: OrganDwarfHeart
   name: dwarf heart
```

Comment:
I'm thinking we should probably just remove the dwarf name designators, since no other species' organs uses distinct names unless the sprite is something very exotic, like slime gas sacs

Reply:
We'll need to resolve if Dwarfs are a human subspecies or not:

yes - should use the same organs
no - should use their own organs outright

Reply:
This is an irrelevant distinction because Reptilians, Mothpeople and Voxes all just use an identical heart, for example. dwarves are the outlier in this
(Now that I think about it, mothpeople should _definitely_ have custom eyes, but that's a separate issue since we need a sprite)

Reply:
See comment below, we're on the same page.

For the time being we could treat "common" organs as "humanlike" just so we have a good word to describe them that isn't just "human".

Reply:
This note is about the name. We don't _actually_ describe any organ as "human" at any point, in a place that is player visible. All we need to do is simply not name them "dwarf organ", just "organ".